PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -365,6 +365,10 @@ def miniAOD_customizeMC(process):
     #also jet flavour must be switched
     process.patJetFlavourAssociation.rParam = 0.4
 
+
+def miniAOD_customizeGEN(process):
+    process.tauGenJets.GenParticles = "prunedGenParticles"
```

Comment:
I think you are right and probably this is not needed either, will make some tests